spyder/plugins/editor/extensions/docstring.py

```diff
@@ -0,0 +1,441 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright © Spyder Project Contributors
+# Licensed under the terms of the MIT License
+# (see spyder/__init__.py for details)
+"""Generate Docstring."""

```

Comment:
Add a blank line here to cleanly separate the ``import``s from the header and docstring (it looks strange to have one line between import sections, and two after, but none before, and doesn't match our convention elsewhere).